c2rust-analyze/src/rewrite/expr/convert.rs

```diff
@@ -224,38 +224,81 @@ impl<'tcx> ConvertVisitor<'tcx> {
             }
 
             mir_op::RewriteKind::MemcpySafe {
-                elem_size,
+                ref elem_ty,
                 dest_single,
+                dest_option,
                 src_single,
+                src_option,
             } => {
                 // `memcpy(dest, src, n)` to a `copy_from_slice` call
```

Comment:
The actual call is `memcpy(dest, src, byte_len)` right?

Reply:
Updated comment in 4808c78